indico/MaKaC/PDFinterface/conference.py

```diff
@@ -78,6 +77,30 @@
 ]
 
 
+# Change reportlab default pdf font Helvetica to indico ttf font,
+# because of better support for international characters.
+# Default font can't be easily changed by setting new value to
+# reportlab.rl_config.canvas_basefontname, because this variable
+# is used directly to initialize state of several reportlab modules
+# during their (first) import.
+def getSansStyleSheet():
+    _font_map = {
+        'Helvetica': 'Sans',
+        'Helvetica-Bold': 'Sans-Bold',
+        'Helvetica-Oblique': 'Sans-Italic',
+        'Helvetica-BoldOblique': 'Sans-Bold-Italic',
+    }
+
+    styles = getSampleStyleSheet()
+    for (name, style) in styles.byName.items():
```

Comment:
I wouldn't object strongly to `.items()`, as it's Python 3-friendly and this is not a method that will be called that often (and I don't think the dict is that large :wink:)